tools/build_file_generator/ThirdPartyDepsTool.java

```diff
@@ -18,39 +18,61 @@
 
 import com.google.startupos.common.CommonModule;
 import com.google.startupos.common.FileUtils;
-import com.google.startupos.common.flags.Flag;
-import com.google.startupos.common.flags.FlagDesc;
-import com.google.startupos.common.flags.Flags;
+import com.google.startupos.tools.buildfilegenerator.Protos.ThirdPartyDeps;
 import dagger.Component;
 
 import javax.inject.Singleton;
 import java.io.IOException;
+import java.net.URI;
+import java.nio.file.FileSystem;
+import java.nio.file.FileSystems;
+import java.nio.file.Files;
+import java.nio.file.Path;
 import java.nio.file.Paths;
+import java.nio.file.StandardCopyOption;
+import java.util.HashMap;
+import java.util.Map;
 
-import com.google.startupos.tools.buildfilegenerator.Protos.ThirdPartyDeps;
-
-/* A tool for generating prototxt file to know which classes each third_party dependency includes
- *
- * Note! Before using this tool, it is necessary to run `./build.sh`
- * to have all third_party deps in `bazel-startup-os/external` folder
+/* A tool for generating the zip-archive with prototxt file inside
+ * to know which classes each third_party dependency includes.
+ * Path to the zip-archive: "//tools/build_file_generator/third_party_deps.zip"
```

Comment:
The path is actually `tools/build_file_generator/third_party_deps.zip`
`//tools/build_file_generator/third_party_deps.zip` is a Bazel target.

Reply:
Done.